huggingface_sb3/load_from_hub.py

```diff
@@ -17,19 +17,19 @@ def load_from_hub(repo_id: str, filename: str) -> str:
         )
 
     try:
-      # Get the model from the Hugging Face Hub
-      model_download_url = hf_hub_url(repo_id=repo_id,
-                                 filename=filename)
-      # Downloading and caching the model on your local disk
-      downloaded_model_file = cached_download(model_download_url,
-                                             library_name="huggingface-sb3",
-                                             library_version="1.0.7")
-    
+        # Get the model from the Hugging Face Hub
+        model_download_url = hf_hub_url(repo_id=repo_id,
+                                        filename=filename)
+        # Downloading and caching the model on your local disk
+        downloaded_model_file = cached_download(model_download_url,
+                                                library_name="huggingface-sb3",
+                                                library_version="1.0.7")
```

Comment:
There is a new method called `hf_hub_download` which does `hf_hub_url` +`cached_download`, so you can simplify all above to

```
downloaded_file = hf_hub_download(
  repo_id=repo_id,
  filename=filename,
  library_name=...
  l
)
```
https://github.com/huggingface/huggingface_hub/blob/3e72bc876d72855445a3c204f483eb761eb91da5/src/huggingface_hub/file_download.py

Reply:
✔️